### PR TITLE
Milestone/open api integration

### PR DESCRIPTION
**Pull request by tomaszmichalak**
Open API configuration for the example project.

## Description
Open API configuration for the example project.

## Upgrade notes (if appropriate)
Upgrade notes should appear in Knot.x Core.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the [code style](/CONTRIBUTING.md#coding-conventions) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

---
I hereby agree to the terms of the Knot.x Contributor License Agreement.

### CODE REVIEW

File: acme-stack/src/main/packaging/conf/application.conf [hunk 1]
```diff
@@ -14,47 +14,154 @@ modules = [
...
 ]
 
-########### Globals ###########
 global {
-  serverPort = 8092
 
-  snippetTagName = script
+  ########### Snippets ###########
+  snippetOptions {
+    # Knot.x snippet HTML tag name. Default is 'script'.
```
Comment (Skejven) [inline acme-stack/src/main/packaging/conf/application.conf]: Just to remember (to-do in another PR): `knotx:snippet` will be default since `2.0`.

File: acme-stack/src/main/packaging/conf/includes/server.conf [hunk 2]
```diff
@@ -1,129 +1,69 @@
...
-  routing.GET.items = [
-    {
-      path = "/content/.*"
-      address = ${global.address.knot.forms}
-      onTransition.next {
-        address = ${global.bridge.address}
-        onTransition.next.address = ${global.address.knot.hbs}
+serverOptions.port = ${global.server.port}
+
+# Open API specification url, see https://vertx.io/docs/apidocs/io/vertx/ext/web/api/contract/openapi3/OpenAPI3RouterFactory.html#createRouterFactoryFromFile-io.vertx.core.Vertx-java.lang.String-io.vertx.core.Handler-
```
Comment (Skejven) [inline acme-stack/src/main/packaging/conf/includes/server.conf]: The url points to `@Deprecated` method.

File: acme-stack/src/main/packaging/conf/application.conf (hunk 1 above)
Comment (pun-ky) [reply acme-stack/src/main/packaging/conf/application.conf]: colon is problematic, why not `knotx-snippet`  ? colon indicates a xml namespace for some IDE's and causes IDE warnings in the end

File: acme-stack/src/main/packaging/sample/content/multiple-forms.html [hunk 3]
```diff
@@ -68,14 +68,14 @@ <h1>{{labelsrepository._result.welcomeInCompetition}}</h1>
...
-      </script>
+      </knotx:snippet>
     </div>
   </div>
   <div class="row">
     <div class="col-md-12">
-      <script data-knotx-knots="form-newsletter,databridge,handlebars"
-              data-knotx-databridge-name-labelsrepository="labelsRepository"
-              data-knotx-databridge-params-labelsrepository='{"path":"/service/labelsRepository.json"}'
+      <knotx:snippet data-knotx-knots="form-newsletter,databridge,handlebars"
```
Comment (pun-ky) [inline acme-stack/src/main/packaging/sample/content/multiple-forms.html]: `knotx-snippet` :)

File: acme-stack/src/main/packaging/conf/application.conf (hunk 1 above)
Comment (Skejven) [reply acme-stack/src/main/packaging/conf/application.conf]: `knotx` will be a separate namespace of Knot.x tags. There will be more types than just `knotx-snippet`. Warnings in the IDE is not a real problem (you can configure IDE for that), sorry :)

Comment (pun-ky) [reply acme-stack/src/main/packaging/conf/application.conf]: it was not only a problem on IDE as I remember. it was impossible to use it in HTL templates.

Comment (tomaszmichalak) [reply acme-stack/src/main/packaging/conf/application.conf]: @pan-ky started discussion about HTL on separate Slack channel
